use-timescale/hypertables/hypertable-crud.md

```diff
@@ -71,34 +70,63 @@ ALTER TABLE conditions
 If the column you are adding has the default value set to `NULL`, or has no
 default value, then adding a column is relatively fast. If you set the default
 to a non-null value, it takes longer, because it needs to fill in this value for
-all existing rows of all existing chunks.
+all existing rows of all existing $CHUNK.
 
 ### Rename a hypertable
 
-You can change the name of a hypertable using the `ALTER TABLE` command. In this
-example, the hypertable is called `conditions`, and is being changed to the new
+You can change the name of a $HYPERTABLE using the `ALTER TABLE` command. In this
+example, the $HYPERTABLE is called `conditions`, and is being changed to the new
 name, `weather`:
 
 ```sql
 ALTER TABLE conditions
   RENAME TO weather;
 ```
 
+### Change a column data type
+
+You can change the data type of a column in a $HYPERTABLE using the `ALTER TABLE`
+command. In this example, the `temperature` column data type is changed from `DOUBLE PRECISION`
+to `NUMERIC`:
+
+```sql
+ALTER TABLE conditions
+  ALTER COLUMN temperature TYPE NUMERIC;
+```
+
+The following restrictions apply:
+
+- You cannot change the type of `segmentby` columns.
```

Comment:
```suggestion
- You cannot change the type of `segmentby` columns.
- You cannot change the type of columns when any chunk of the hypertable uses columnstore
```

Reply:
Added in a sightly different wording and linked to the dedicated page about it